src/types/impl-trait.md

```diff
@@ -53,6 +65,16 @@ This includes generic arguments for the return type or any const generics.
 >
 > Therefore, changing the function signature from either one to the other can constitute a breaking change for the callers of a function.
 
+## Abstract types
+
+> Note: these are also called "existential types" or "opaque types".
```

Comment:
Style-wise, I suggest using italics instead of quotes when introducing terms or marking key phrases. I realize the rest of the text isn't always consistent with that, but I think that is the generally accepted way to format these. (There's a few other examples below that could be changed.)

```suggestion
> Note: these are also called *existential types* or *opaque types*.
```